src/store/reducers/settings.js

```diff
@@ -19,6 +19,7 @@ const initialState = JSON.parse(localStorage.getItem('settings')) || {
   isHarwareWalletConnected: false,
   isRequestHowItWorksDisable: false,
   statistics: false,
+  isTermsOfUse: true,
```

Comment:
Naming the flag `isTermsOfUse` IMO doesn't clearly say if `true` means terms were accepted or `false` means terms were accepted. I suggest:

```suggestion
  areTermsOfUseNotAccepted: true,
```

Reply:
change done.